web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css

```diff
@@ -75,13 +75,29 @@
 
 /* This css class is added on the projects view (admin/structure/views/view/projects) */
 .projects-view {
-  form {
-    display: flex;
+  .exposed-filter {
+    #edit-actions {
+      grid-column: span 1 / span 1;
+      margin-bottom: 1.25rem;
```

Comment:
Can we reuse some spacing attributes from tailwind? Or create som custom properties resembling the tailwind equivalents, and the use them, so we ensure consistency.

Reply:
I don't have immediate access to the templates for these particular buttons, so we have to use css nesting. I'm not sure if it's possible to extend or somehow reuse the tailwind classes